src/pages/patientView/structuralVariant/PatientViewStructuralVariantTable.tsx

```diff
@@ -85,18 +85,23 @@ export default class PatientViewStructuralVariantTable extends React.Component<
             if (numSamples >= 2) {
                 columns.push({
                     name: 'Samples',
-                    render: (d: StructuralVariant[]) =>
-                        TumorColumnFormatter.renderFunction(
+                    render: (d: StructuralVariant[]) => {
+                        return TumorColumnFormatter.renderFunction(
                             d.map(datum => ({
                                 sampleId: datum.sampleId,
-                                entrezGeneId: datum.site1EntrezGeneId,
+                                entrezGeneId: [
```

Comment:
@inodb safe to assume they all have site1 and site2?

Reply:
Some might have only one gene (a lot of them can be outside of genes). For data import at least either site 1 or site 2 needs to be specified